app/graphql/loaders/installments_loader.rb

```diff
@@ -1,5 +1,19 @@
 class Loaders::InstallmentsLoader < Loaders::FancyLoader
   from Installment
+  modify_query ->(query) {
+    release_rank = Loaders::FancyLoader::RankQueryGenerator.new(
+      :release_order,
+      @find_by,
+      table
+    ).arel
```

Comment:
This might look nicer if you switch to keyword arguments 🤔 Not 100% sure though, feel free not to

```ruby
release_rank = Loaders::FancyLoader::RankQueryGenerator.new(
  column: :release_order,
  table: table,
  partition_by: @find_by
).arel
```